### PR TITLE
Ephys Improvements

### LINKED ISSUE #971: small ephys fixes
- [ ] make `level` in `Anaesthetic` an optional field
- [ ] in `Session` make the reward consumed (both for the full session and per stimulus epoch) have a default unit of mL. This is updated the title of the field and making the unit default setting mL.

### PR DESCRIPTION
**Pull request by Sun-flow**
closes #971

`Anaesthetic.level` -> `Optional`
`Session.reward_consumed_unit` -> new default: mL

### CODE REVIEW

File: src/aind_data_schema/core/procedures.py [hunk 1]
```diff
@@ -277,7 +277,7 @@ class Anaesthetic(AindModel):
     type: str = Field(..., title="Type")
     duration: Decimal = Field(..., title="Duration")
     duration_unit: TimeUnit = Field(default=TimeUnit.M, title="Duration unit")
-    level: Decimal = Field(..., title="Level (percent)", ge=1, le=5)
+    level: Optional[Decimal] = Field(..., title="Level (percent)", ge=1, le=5)
```
Comment (saskiad) [inline src/aind_data_schema/core/procedures.py:280]: default needs to be None for optional

**Review by saskiad — CHANGES_REQUESTED** [review]
fix the default!

**Review by dyf — CHANGES_REQUESTED** [review]
accidentally approved before, see comment above

File: src/aind_data_schema/core/procedures.py [hunk 2]
```diff
@@ -277,7 +277,7 @@ class Anaesthetic(AindModel):
     type: str = Field(..., title="Type")
     duration: Decimal = Field(..., title="Duration")
     duration_unit: TimeUnit = Field(default=TimeUnit.M, title="Duration unit")
-    level: Decimal = Field(..., title="Level (percent)", ge=1, le=5)
+    level: Optional[Decimal] = Field(None, title="Level (percent)", ge=1, le=5)
```
Comment (saskiad) [inline src/aind_data_schema/core/procedures.py:280]: add the default= here as well

**Review by saskiad — CHANGES_REQUESTED** [review]
we want the default= in all the places that there's a default value. It improves the understandability of the schema.

**Review by saskiad — APPROVED** [review]

**Review by dyf — APPROVED** [review]

**Pull request merged**